release-notes.md

```diff
@@ -1,3 +1,50 @@
+
+# Release Notes 2.0.0a9 (Dec 9th, 2021)
+
+We are happy to announce another small increment towards our plan to a stable 2.0 version.
+
+The following 4 feature increments should be highlighted:
+
+**The UI is now shipped as part of every release**<br/>
```

Comment:
```suggestion
**The UI is now shipped as part of every release**
```
We shouldn't need `<br/>` here?

Reply:
I wanted to make every highlighted feature standing out on one line.